Frameworks/AddressBook/ABAddressBookManagerInternal.mm

```diff
@@ -172,22 +200,36 @@ - (id)init {
     return self;
 }
 
+@dynamic contactStore;
```

Comment:
I would be happy enough to let autosynthesis do its thing, except that in this case I want the property and the synthesized field to have different types.